airbyte-integrations/connectors/source-google-search-console/unit_tests/test_migrations/test_config_migrations.py

```diff
@@ -16,6 +18,7 @@
 # BASE ARGS
 CMD = "check"
 TEST_CONFIG_PATH = "unit_tests/test_migrations/test_config.json"
+BACKUP_CONFIG_PATH = "unit_tests/test_migrations/test_config.json.bak"
```

Comment:
nit: I recommend using something like `str(Path(__file__).parent / "test_config.json.bak")` to avoid having to deal with differences in the current working directory